frontend/simple/views/Invite.vue

```diff
@@ -86,105 +33,109 @@
+    GroupInvitees
+  },
   data () {
-    // TODO: https://github.com/okTurtles/group-income-simple/issues/297
     return {
-      searchUser: null,
-      members: [],
-      userErrorMsg: '',
-      invited: false,
-      errorMsg: null
+      form: {
+        invitees: [],
+        success: false,
+        errorMsg: null
+      }
     }
   },
-  methods: {
-    async add () {
-      if (!this.searchUser) return
-
-      if (this.searchUser === this.$store.state.loggedIn.name) {
-        this.userErrorMsg = L('Invalid User: Cannot Invite One\'s self')
-        return
-      } else {
-        this.userErrorMsg = ''
-      }
-
-      try {
-        const contractId = await namespace.lookup(this.searchUser)
-        const state = await latestContractState(contractId)
-        if (!this.members.find(member => member.state.attributes.name === this.searchUser)) {
-          this.members.push({ state, contractId })
-        }
-        this.searchUser = null
-        this.userErrorMsg = ''
-      } catch (err) {
-        console.log(err)
-        this.userErrorMsg = L('Invalid User')
-      }
+  computed: {
+    isProposal () {
+      return this.memberCount() >= 3
     },
-    remove (index) {
-      this.members.splice(index, 1)
+    ...mapState([
+      'currentGroupId',
+      'loggedIn'
+    ]),
+    ...mapGetters([
+      'currentGroupState',
+      'memberCount'
+    ])
+  },
+  methods: {
+    updateInvitees (payload) {
+      this.form.errorMsg = null
+      Object.assign(this.form, payload.data)
     },
     async submit () {
       try {
-        this.errorMsg = null
-        // TODO: as members are successfully invited display in a
-        // seperate invitees grid and add them to some validation for duplicate invites
-        for (let member of this.members) {
-          // We need to have the latest mailbox attribute for the user
-          const mailbox = await backend.latestHash(member.state.attributes.mailbox)
-          const sentDate = new Date().toString()
+        this.form.errorMsg = null
+        const groupId = this.currentGroupId
+        const groupName = this.currentGroupState.groupName
+
+        for (let member of this.form.invitees) {
+          let mailbox = member.state.attributes.mailbox
+          let memberName = member.state.attributes.name
 
-          // We need to post the invite to the users' mailbox contract
-          const invite = new Events.HashableMailboxPostMessage(
-            {
-              from: this.$store.getters.currentGroupState.groupName,
-              headers: [this.$store.state.currentGroupId],
-              messageType: Events.HashableMailboxPostMessage.TypeInvite,
-              sentDate
-            },
-            mailbox
-          )
-          await backend.publishLogEntry(member.state.attributes.mailbox, invite)
+          if (this.isAlreadyInvited(member)) {
+            template(L('Failed to invite users: member {name} is already invited'),
+              { name: memberName })
```

Comment:
This line doesn't appear to be doing anything.

Reply:
🙈 yeah I just forgot the first half